deployment/def/powerbi.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: powerbi-deployment
+  labels:
+    system: a01
+    group: base
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: powerbi
+    spec:
+      containers:
+      - name: powerbi-svc
+        command: ["powerbi"]
+        image: adxautomationbase.azurecr.io/powerbi:0.1.0
+        ports:
+        - containerPort: 80
+        env:
+        - name: A01_POWERBI_CLIENT_ID
+          valueFrom:  
+            secretKeyRef:
+              name: email
+              key: powerbi.client.id
+        - name: A01_POWERBI_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: email
+              key: username
+        - name: A01_POWERBI_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: email
+              key: password
+      imagePullSecrets:
+      - name: adxautomationbase-registry
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: powerbi-internal-svc
```

Comment:
Rename it. `PowerBI` is the implementation. The actual function is "dashboard". Name the service for what it does instead of how it is built.

Reply:
Fixed. Also fixed on the dispatcher side